internal/connectors/db/process_result_set.go

```diff
@@ -206,3 +207,92 @@ func ReadOperationFromResultSet(res result.Result) (types.Operation, error) {
 
 	return &types.GenericOperation{ID: operationId}, nil
 }
+
+func ReadBackupScheduleFromResultSet(res result.Result) (*types.BackupSchedule, error) {
+	var (
+		ID               string
+		containerID      string
+		databaseName     string
+		databaseEndpoint string
+		active           bool
+
+		crontab string
+
+		initiated               *string
+		createdAt               *time.Time
+		name                    *string
+		ttl                     *time.Duration
+		sourcePaths             *string
+		sourcePathsToExclude    *string
+		recoveryPointObjective  *time.Duration
+		lastBackupId            *string
```

Comment:
`lastBackupId` -> `lastBackupID` and  `lastSuccessfullBackupId` -> `lastSuccessfullBackupID`